src/main/java/org/jenkinsci/plugins/github/config/GitHubServerConfig.java

```diff
@@ -138,6 +131,7 @@ public String getApiUrl() {
 
     /**
      * @see #isUrlCustom(String)
+     * @deprecated
      */
     @Restricted(NoExternalUse.class)
     public boolean isCustomApiUrl() {
```

Comment:
Just remove if it not used anymore, it was only for UI checkboxes because of pure jelly forms.